modules/gltf/SCsub

```diff
@@ -7,7 +7,7 @@ env_gltf = env_modules.Clone()
 
 # Godot source files
 env_gltf.add_source_files(env.modules_sources, "*.cpp")
-env_gltf.add_source_files(env.modules_sources, "extensions/*.cpp")
 env_gltf.add_source_files(env.modules_sources, "structures/*.cpp")
+SConscript("extensions/SCsub")
```

Comment:
What's the use for this? It's not done for structures nor editor, if there's no custom config to set for this folder / a sub env, I don't see how it makes things better.

Reply:
In isolation it's indeed not better, but it's planning for future PRs where there is more complex logic. In the future I'm planning to have `extensions/` to have subfolders in it. If you want I can remove it from this PR, and instead we can introduce this SCsub when it becomes useful.

Reply:
I guess it's fine if you're confident that these future PRs will actually be merged as you envision them. But generally speaking yes, it's best to leave out such future proofing and do the change when it's actually useful.